### DISCUSSION TITLE
[THEME] VS Code Dark Plus

### DISCUSSION THREAD

**Discussion by greengeek** [TriliumNext/Trilium#3485] [Show and tell]
I really like Visual Studio Code's default dark theme called Dark+.   While others have created versions of this theme, I have not been able to get them to work with the latest version of Trilium Server.  I decided to give it a try.  Please let me know if you have any comments or suggestions. Thanks!

The theme can be found here https://github.com/greengeek/trilium-vscode-dark-theme

Here are some screenshots.  

<img src="https://raw.githubusercontent.com/greengeek/trilium-vscode-dark-theme/main/screenshot1.png" />
<img src="https://raw.githubusercontent.com/greengeek/trilium-vscode-dark-theme/main/screenshot2.png" />
<img src="https://raw.githubusercontent.com/greengeek/trilium-vscode-dark-theme/main/screenshot3.png" />

**Comment by greengeek** [comment]
Updated with some minor changes:

Added additional spacing above the page title to make it easier to read and to put it closer to being at the same level as the root menu heading in the left bar.
Dimmed the controls below the page title to make the page feel less busy. 
Slightly lightened the hyperlinks, so they are the same color as the icons on the left to make them easier to read and add a bit more contrast.  (not shown in current screenshots).

**Comment by greengeek** [comment]
Updated with a few additional minor changes.

- Tightened spacing in left tree bar.
- Smoothed scrolling on the left tree bar.
- Adjusted padding to the title on right bar. 
- Adjusted background color in filler bar up top.

Updated screenshots.

**Comment by Nriver** [comment]
I've added your theme into https://github.com/Nriver/awesome-trilium. If you have more to share in the future, welcome to update the list.

**Comment by tylerlmz1** [comment]
Really love this theme, particularly how it makes the tree view more compact, that I can see more items in my tree view.

Overall very nice, thanks
